connector/src/main/java/org/geysermc/connector/entity/LivingEntity.java

```diff
@@ -57,6 +60,16 @@ public LivingEntity(long entityId, long geyserId, EntityType entityType, Vector3
     @Override
     public void updateBedrockMetadata(EntityMetadata entityMetadata, GeyserSession session) {
         switch (entityMetadata.getId()) {
+            case 7: // blocking
+                byte xd = (byte) entityMetadata.getValue();
+
+                //blocking gets triggered when using a bow, but if we set USING_ITEM for all items, it may look like
+                //you're "mining" with ex. a shield.
+                boolean isUsingShield = (getHand().getId() == ItemRegistry.SHIELD_BEDROCK ||
```

Comment:
The reason I have not merged this yet is that I don't like the idea of having two static variables for the shield... I personally would prefer one static variable, and then translating that value if we have to. This is more than open to discussion.